third_party/compute_library/BUILD

```diff
@@ -13,7 +13,7 @@ cc_library(
 )
 
 _COMPUTE_LIBRARY_DEFINES = [
-    "ARM_COMPUTE_CPP_SCHEDULER",
+    "ARM_COMPUTE_OPENMP_SCHEDULER",
```

Comment:
Does this mean we are only using OpenMP for scheduler or for threading as well? Could you please explain what this would mean for making TF-oneDNN-aarch64 use TF's Eigen thread pool?

Reply:
Hi @penpornk,

This change only affects the 'non threadpool' build, which you'll get with `--config=mkl_aarch64` - previously this would have used oneDNN's OpenMP runtime, but with the ACL primitives using ACL's CPP schedular.
The 22.08 release fixes a bug in ACL's OpenMP scheduler, so this PR updates the build to use this in preference to the default, CPP, scheduler - using OpenMP for oneDNN and ACL.

This doesn't affect the 'threadpool' build (`--config=mkl_aarch64`, used in the CI/CD builds (#56924), for which the oneDNN build will leverage ACL's support for 'custom' (i.e. external) schedulers to use Eigen's threadpool throughout).

So, this change doesn't impact the 'threadpool' build we're using for the wheels, but if someone does a build with an 'OpenMP' version of oneDNN, then that's what they'll get (no ACL pthreads).

Reply:
Thank you for the explantion!

Just for future reference, I think you meant `--config=mkl_aarch64_threadpool` here:
> This doesn't affect the 'threadpool' build (--config=mkl_aarch64, used in the CI/CD builds (#56924), for which the oneDNN build will leverage ACL's support for 'custom' (i.e. external) schedulers to use Eigen's threadpool throughout).

Reply:
You're right! Sorry, updated the original comment just to be sure.